environment.yml

```diff
@@ -14,3 +14,4 @@ dependencies:
   - r-testthat >=3.0.0
 # Others
   - r-lintr
+  - r-osmdata
```

Comment:
Maybe put `r-osmdata` right after `r-sfnetworks` in the list, as it is part of the "Suggests" section?